scripts/upgrade.d/upgrade.15.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Last available nextcloud version
+next_version="16.0.0"
```

Comment:
```suggestion
next_version="15.0.11"
```

Reply:
Hm, thanks for that, but shouldn't we make a dedicated 15.0.11 PR and keep that NextCloud 16 (and even 17!) upgrade PR open, just to indicated we've already though about it for people passing by...? :-)

Reply:
You're right, I'll update these suggestions to keep the current nextcloud version (15.0.10)